### ISSUE TITLE
Render as you Fetch?

### ISSUE THREAD

**Issue by maraisr** [atlassian-labs/react-resource-router#30]
Do these resources go in-flight when the route transitions, but parallel to fetching async code? Definitely keen to try this out!

**Comment by rohan-deshpande** [comment]
Yep. Resources are requested immediately on transitioning into the next route as long as they have not been previously cached and are outside the `maxAge` value.

**Comment by maraisr** [comment]
I won't then if this plays nicely into Relay's EntryPoints and preloaded queries. We use relay, and have its render as you fetch working, but preloaded fire for all routes naturally. Which led me here.
